Changes

```diff
@@ -23,6 +23,8 @@ NEXT_RELEASE:
 OCamlbuild 0.9.0 (19 Jan 2016):
 -------------------------------
 
+- #44: When ocamlbuild discover a cycle in the dependencies, the
```

Comment:
Fixed, thanks. (Github seems confused about which patches are part of this PR, this one actually isn't, but that's fine)